src/libraries/System.Private.CoreLib/src/System/Reflection/SignatureGenericParameterType.cs

```diff
@@ -26,6 +26,7 @@ protected SignatureGenericParameterType(int position)
         public sealed override bool IsGenericParameter => true;
         public abstract override bool IsGenericMethodParameter { get; }
         public sealed override bool ContainsGenericParameters => true;
+        protected sealed override bool IsValueTypeImpl() => false;
```

Comment:
Would it be better if this continues to throw? IsValueType for regular generic parameters is complicated. It returns whether the generic parameter is constrained as `struct`.

Reply:
I will change it to throw - thanks. We can always add support later if necessary.